src/app/views/query-response/pivot-items/pivot-items.tsx

```diff
@@ -23,7 +24,10 @@ import { Snippets } from '../snippets';
 export const getPivotItems = (properties: any) => {
 
   const { headers, body, mode, sampleQuery } = properties;
-  const resultComponent = displayResultComponent(headers, body);
+  let { height } = properties;
```

Comment:
this makes height and any type so we lost all typing support. just pass the height into the convert directly.